styles.css

```diff
@@ -91,9 +91,175 @@ section ul li:first-child {
   font-size: 27px;
 }
 
+.sections {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  padding: 2rem;
+  max-width: 378px;
+}
+
+.colorh4 {
+  color: #344563;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.canopy {
+  list-style-type: disc;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: baseline;
+  margin-bottom: 1rem;
+}
+
+li span img {
+  background-color: #7a869a;
+  color: #7a869a;
+}
+
+.text {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+section .canopy ul li {
+  font-weight: 600;
+  padding: 0 14px;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.colorh2 {
+  font-style: normal;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin: 1.2rem 0;
+}
+
+.languages {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  margin: 14px;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 7px 10px;
+}
+
+.languages:first-child {
+  padding-left: 10px;
+  margin-left: 0;
+}
+
+.seeproject {
+  height: 24px;
+  max-width: -moz-fit-content;
+  max-width: fit-content;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background: #fff;
+  border-radius: 16px;
+}
+
+#aboutme {
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
[OPTIONAL] I would heavily advise against using id selectors in your CSS files for styling as ids are unique to individual elements and the styles cannot be reused but have to be rewritten for every other element that uses those styles 👍. It is always a good idea to save the ids for manipulation in JavaScript. 👍 You can read more about it in this [stack overflow thread](https://stackoverflow.com/questions/8279132/why-shouldnt-i-use-id-selectors-in-css#:~:text=It%20is%20not%20advisable%20to,always%20advisable%20to%20use%20class.)